### ISSUE TITLE
Fetch multiple attributes from Historical Data

### ISSUE THREAD

**Issue by sakshi2809sharma** [telefonicaid/fiware-sth-comet#442]
Is there any REST API to fetch multiple attributes from Historical Data:

I want to represent the updated Historical data in tabular format. But when I update the data using below REST API (POST), only the attributes in request body are updated:
https://<sth-host>:<sth-port>/orion/v1.0/updateContext

The remaining attributes are not updated. So, it gets difficult to map all the attributes at a particular timestamp.
The below REST API (GET) gives updated value of only one attribute at a time:
http://<sth-host>:<sth-port>/STH/v1/contextEntities/type/<entityType>/id/<entityId>/attributes/<attrName>?filetype=csv

**Comment by fgalan** [comment]
> Is there any REST API to fetch multiple attributes from Historical Data?

I don't think so. By design, the API is oriented to single-attribute queries.

The problem implementing such functionalyt is that attribute update times are not necesarily aligned, so at the end, it would be a sum of independent histories (which is the same the client can get with the current API, using several call to it, one per attribute to retrieve).
